src/templates/find_usages.html

```diff
@@ -1,4 +1,4 @@
-<table class="table table-condensed">
+<table class="table table-hover">
```

Comment:
it felt too compact. It seemed good for when you couldn't interact with the table, but it's nice to have a bit of spacing when you're clicking on something.

That said I don't have a strong feeling on this at all and will do whatever you think is best